### PR TITLE
[Download] Small fixes/updates

### PR DESCRIPTION
**Pull request by chejennifer**
- update text for place selector, no variables selected message, and update button
- add additional option for selecting all available dates
- fix bug where preview table shows loading and then disabled when options change
- add info text
Screen Shot 2022-08-05 at 11 39 31 AM

### CODE REVIEW

File: static/js/tools/download/info.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          thousands of statistical variables to choose from, arranged in a
+          topical hierarchy.
+        </li>
+        <li>
+          Choose whether you want the data for the latest date, all available
+          dates, or a custom date range.
+        </li>
+      </ol>
+      <p>
+        Data will be downloaded as a .csv file where each row will contain the
```
Comment (shifucun) [inline static/js/tools/download/info.tsx:46]: will contain -> contains

File: static/js/tools/download/info.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              <th>Date:[Variable2]</th>
+              <th>Value:[Variable2]</th>
+              <th>Source:[Variable2]</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>geoId/01</td>
+              <td>Alabama</td>
+              <td>01-2020</td>
```
Comment (shifucun) [inline static/js/tools/download/info.tsx:67]: should date be 2020-01?

**Review by shifucun — APPROVED** [review]
Also the npm test has been timeout. Worth to see what cause this.

File: static/js/tools/download/info.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/download/info.tsx]: done

File: static/js/tools/download/info.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/tools/download/info.tsx]: good catch, done

**Comment by chejennifer** [comment]
Thanks for the review! I've been noticing the timeouts too and I've been meaning to find time to look into them, will try to do it this week

**Pull request merged**